api/preprint_providers/views.py

```diff
@@ -20,7 +24,7 @@
 
 from api.preprints.permissions import PreprintPublishedOrAdmin
 
-class PreprintProviderList(JSONAPIBaseView, generics.ListAPIView, ODMFilterMixin):
+class PreprintProviderList(JSONAPIBaseView, generics.ListAPIView, DjangoFilterMixin):
```

Comment:
Not that I can tell -- they both just provide `get_query_from_request`, and require overriding `get_default_(odm|django)_query`. The rest of both mixins are internal, looks like.